submet/metrics.py

```diff
@@ -1,6 +1,29 @@
 import numpy as np
+import scipy.spatial.distance as ssd
+from scipy.stats import spearmanr
 
```

Comment:
spearmanr is still susceptible to issues related to columns of all NaN -- we can either create a pull request on the scipy package, fixing this issue, or we can check for columns of all **NaNs** and replace them with **Inf**